ocs_ci/helpers/helpers.py

```diff
@@ -203,7 +203,7 @@ def create_pod(
             ] = pvc_name
 
     if interface_type == constants.CEPHBLOCKPOOL and raw_block_pv:
-        if pod_dict_path in [constants.FEDORA_DC_YAML, constants.FIO_DC_YAML]:
+        if pod_dict in [constants.FEDORA_DC_YAML, constants.FIO_DC_YAML]:
             temp_dict = [
```

Comment:
Why did you change the dictionary name `pod_dict_path`?

Reply:
This change was suggested by Jilju for the issue- 2524.

Reply:
if this is to resolve issue make sure to add Fixes in PR

Reply:
thanks @prsurve